Integrations/FeedTAXII/README.md

```diff
@@ -26,6 +26,45 @@ Ingests indicator feeds from TAXII 1.x servers.
     * __First Fetch Time__: The time interval for the first fetch (retroactive). <number> <time unit> of type minute/hour/day. For example, 1 minute, 12 hours, 7 days.
 4. Click __Test__ to validate the URLs, token, and connection.
 
+## Step by step configuration
+---
+As an example, we'll be looking at the public TAXII threat intelligence feed by Abuse_ch accessible via _Hail a TAXII_. These are the feed instance configuration parameters for our example.
+
+**Indicator Reputation** - Because this is just an example, we can leave the default value. Ordinarily you would set the reputation based off the specific feed's information about what type of indicators they are returning, e.g., whether they are good or bad.
+
+**Source Reliability** - Because this is just an example, we can leave the default value. Ordinarily you would set the reliability equal to how much you trusted the feed's source.
+
+**Indicator Expiration Method** - For this example, we can leave the default value here. Ordinarily you would set the value according to the type of feed you were fetching from. As an example, let us say that you are a customer of a Cloud Services provider and you want to add the URLs from which that provider serves up a lot of the services you use to your network firewall exclusion list. Assuming that that same Cloud Services provider maintains an up-to-date feed of the URLs from which they currently provide service, you would probably want to configure a feed integration instance with this parameter set to `Expire indicators when they disappear from feed` so that you don't continue to mark a given URL with a `Good` reputation after it is no longer being used by your Cloud Services provider.
+
+**Feed Fetch Interval** - For this example, we can leave the default value here.
+
+**Discovery Service** - Enter `http://hailataxii.com/taxii-discovery-service`.
+
+**Collection** - Enter `guest.Abuse_ch`
+
+**Subscription ID** - No need to enter a value here for this example since the TAXII server we are addressing does not require it so we'll leave it blank.
+
+**Username** - Enter `guest`.
+
+**Password** - Enter `guest`.
+
+**Request Timeout** - Let's increase the number to `80`  seconds since the request may take a while complete.
```

Comment:
note the extra whitespace between `80` and `seconds`

Reply:
👍 fixed.